src/libs/tmdk/src/lib/component-type.ts

```diff
@@ -131,6 +124,10 @@ async function waitForComponentTypeActive(
       console.log(`${componentTypeId} still not found...`);
     }
   }
+  throw new Error(
+    `Timed out: too many attempts for componentId: ${componentTypeId}\n
```

Comment:
Nit: add description of what we attempted, ie `too many attempts for GetComponentType for componentId...`